velox/docs/configs.rst

```diff
@@ -703,6 +703,11 @@ Each query can override the config by setting corresponding query session proper
      - integer
      - 1024
      - Batch size used when writing into Parquet through Arrow bridge.
+    * - hive.parquet.writer.created-by
```

Comment:
This is not aligned properly.
You can see from "View file" that this table is now empty because of the misalignment.